jsk_pcl_ros/launch/plane_extraction.launch

```diff
@@ -0,0 +1,109 @@
+<launch>
```

Comment:
can you move this file to `jsk_pcl_ros/sample`?

Reply:
can you also rename to `sample_plane_extraction.launch`?

Reply:
or you can split this file to `launch/plane_extraction.launch` and `sample/sample_plane_extraction.launch`.

`launch/plane_extraction.launch` is normally same as this exluding `rosbag` and `rviz`.
and `sample/sample_plane_extraction.launch` includes `launch/plane_extraction.launch` and `rosbag` and `rviz` nodes.